x-pack/filebeat/module/fortinet/firewall/manifest.yml

```diff
@@ -0,0 +1,21 @@
+module_version: 1.0
+
+var:
+  - name: syslog_host
+    default: localhost
+  - name: tags
+    default: [fortinet-firewall]
+  - name: syslog_port
+    default: 9001
+  - name: input
+    default: syslog
+  - name: log_level
```

Comment:
Seems to me that this variable is not used, just carried over from using Cisco's module as a base.